src/index.html

```diff
@@ -11,7 +11,131 @@
     <link rel="stylesheet" href="./style.css">
   </head>
   <body>
-    <h1>HTML Form</h1>
+    <form action="https://mate-academy-form-lesson.herokuapp.com/create-application" method="post">
+      <fieldset class="fieldset">
+        <legend>Personal information</legend>
+        <label class="form_field">Surname:
+          <input
```

Comment:
```suggestion
        <label class="form_field">
          Surname:
          <input
```
make sure you fix it everywhere, not just here